contracts/sfc/Staker.sol

```diff
@@ -566,23 +566,23 @@ contract Stakers is Ownable, StakersConstants {
     event ClaimedValidatorReward(uint256 indexed stakerID, uint256 reward, uint256 fromEpoch, uint256 untilEpoch);
 
     // Claim the pending rewards for a given stakerID (sender)
-    // _fromEpoch is starting epoch which rewards are calculated (including). If 0, then it's lowest not claimed epoch
     // maxEpochs is maximum number of epoch to calc rewards for. Set it to your chunk size.
     //
     // may be already deactivated, but still allowed to withdraw old rewards
-    function claimValidatorRewards(uint256 _fromEpoch, uint256 maxEpochs) external {
+    function claimValidatorRewards(uint256 maxEpochs) external {
```

Comment:
can claim rewards partially?

Reply:
Of course, number of non-claimed epochs may be too great to calculate all rewards in one call